app/controllers/users/phones_controller.rb

```diff
@@ -107,9 +109,9 @@ def handle_successful_delete
       create_user_event(:phone_removed)
     end
 
-    def set_phone_id
```

Comment:
No need for a whole method here that just checks and sets the user_session phone_id